packages/xdl/src/detach/AndroidShellApp.js

```diff
@@ -546,13 +546,6 @@ export async function runShellAppModificationsAsync(context, sdkVersion, buildMo
     path.join(shellPath, 'app', 'build.gradle')
   );
 
-  // Push notifications
-  await regexFileAsync(
-    '"package_name": "host.exp.exponent"',
-    `"package_name": "${javaPackage}"`,
-    path.join(shellPath, 'app', 'google-services.json')
-  ); // TODO: actually use the correct file
-
```

Comment:
Maybe add a note in the PR description that you've made this change purposely. I mean you didn't mention anything about changing the behavior of filling in `package_name`.

Reply:
Good suggestion, thank you!